packages/headless/src/controllers/facets/facet/headless-facet.ts

```diff
@@ -131,6 +134,15 @@ export function buildFacet(engine: SearchEngine, props: FacetProps): Facet {
       );
     },
 
+    toggleExclude(selection) {
```

Comment:
Additionally, we will need to work on breadcrumb (`headless-breadcrumb-manager`), since it currently only verify that a value is `selected` to clear them from the filters.

All these comments can be (should be, actually) done in another ticket/PR in order to split the workload and code to review.

I'm mostly pointing these out so that they get on your radar.

Reply:
Created all of the necessary tickets here:
https://coveord.atlassian.net/browse/KIT-2548

I suspect that this PR will be the largest for the Headless changes 😅